### ISSUE TITLE
Ability to control whitespace handling when appending data to fields

### ISSUE THREAD

**Issue by glutanimate** [glutanimate/batch-editing#10]
There are use cases where the defaults of adding a newline between new and old content in regular mode, or adding a space in the same position in HTML mode, are not desirable (see [this report](https://anki.tenderapp.com/discussions/add-ons/36284-batch-edit-is-inserting-spacing-and-new-lines-where-it-shouldnt) for instance). To cover those use cases, a toggle to not include any separating whitespace could be useful.

**Comment by Apetited** [comment]
I would suggest whitespace or newlines could be inserted by the user as desired in the same box where their text is written. This would account for all use cases without the need for any new features.

**Comment by glutanimate** [comment]
Yes, but for the large majority of users adding that whitespace automatically means one less thing to worry about. More importantly though, fundamentally changing the way the add-on behaves at this point would leave the existing user-base confused. So it's not something I'm willing to do.
